File: SkaiLab.Invoice/ClientApp/src/app/models/ProfitAndLostSummary.ts
export class ProfitAndLostSummaryToal{
    name:string;
    values:number[];
    constructor(){
        this.name="";
        this.values=[];
    }
}
export class ProfitAndLostSummaryRow{
    name:string;
    values:number[];
    constructor(){
        this.name="";
        this.values=[];
    }
}
export class ProfitAndLostSummaryRowHeader{
    name:string;
    profitAndLostSummaryRows:ProfitAndLostSummaryRow[];
    constructor(){
        this.name="";
        this.profitAndLostSummaryRows=[];
    }
}
export class ProfitAndLostSummary{
    headers:string[];
    profitAndLostSummaryRowHeaders:ProfitAndLostSummaryRowHeader[];
    profitAndLostSummaryToal:ProfitAndLostSummaryToal;
    constructor(){
        this.headers=[];
        this.profitAndLostSummaryRowHeaders=[];
        this.profitAndLostSummaryToal=new ProfitAndLostSummaryToal();

    }
}

File: SkaiLab.Invoice.Models/Report/ProfitAndLostSummary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SkaiLab.Invoice.Models.Report
{
    public class ProfitAndLostSummary
    {
        public List<string> Headers { get; set; }
        public List<ProfitAndLostSummaryRowHeader> ProfitAndLostSummaryRowHeaders { get; set; }
        public ProfitAndLostSummaryTotal ProfitAndLostSummaryToal { get; set; }
    }
    public class ProfitAndLostSummaryRowHeader
    {
        public string Name { get; set; }
        public List<ProfitAndLostSummaryRow> ProfitAndLostSummaryRows { get; set; }
        public decimal Total { get; set; }
    }
    public class ProfitAndLostSummaryRow
    {
        public string Name { get; set; }
        public List<decimal> Values { get; set; }
    }
    public class ProfitAndLostSummaryTotal
    {
        public string Name { get; set; }
        public List<decimal> Values { get; set; }
    }
}
